src/files/castleclientserver.pas

```diff
@@ -0,0 +1,737 @@
+{
+  Copyright 2018 Benedikt Magnus.
+
+  This file is part of "Castle Game Engine".
+
+  "Castle Game Engine" is free software; see the file COPYING.txt,
+  included in this distribution, for details about the copyright.
+
+  "Castle Game Engine" is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+  ----------------------------------------------------------------------------
+}
+
+unit castleclientserver;
+
```

Comment:
Write it CamelCase, `unit CastleClientServer;`

(Note: The filename should stay lowercase, `castleclientserver.pas`, to be reliably found by FPC on case-sensitive filesystems. But the unit name is better CamelCase -- looks nicer, and makes code completion work :)